1-Vanilla/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```

Comment:
in general great PR! the code is very readable and the logic is split into small and reusable parts 👏
few functional / bugs I've noticed:

- make sure "enter" key also adds a new todo (not just clicking the button)
- Clear input after a new todo is added
- Uncheck don’t Clear the strike through
- The edit is not clear. You take what's currently in the input.. think of a better UX